### ISSUE TITLE
JuliaCon 2025

### ISSUE THREAD

**Issue by rayegun** [cormullion/graphics#33]
Hi Cormullion :),

Are you interested in starting on the 2025 templates and graphics? 

CC: @ranjanan

Best,
Raye

**Comment by cormullion** [comment]
Yes I'd love to contribute to JuliaCon2025 in some/any way - I can't contribute much to Julia code itself, so having a minor role is very rewarding for me... :)

By the way, I was thinking the JuliaCon website could do with an overhaul - unfortunately I'm not much of a web developer (I can just criticize from a safe distance though...)

**Comment by rayegun** [comment]
Do you have any thoughts on a JuliaCon Global 2025 logo? Something Pittsburgh specific? Cc @rkurchin

**Comment by cormullion** [comment]
I must be honest and say I know nothing about Pittsburgh... So any hints or suggestions would be welcome... !

**Comment by cormullion** [comment]
> Pittsburgh is defined by many things, most notably its colors of Black & Gold. (https://pittsburghpa.gov/pittsburgh/flag-seal)

**Comment by rkurchin** [comment]
Black and gold for sure! Some other notable visuals around the city include:
* [Three Sisters](https://www.google.com/search?sca_esv=abe1209c8be175c1&q=pittsburgh+three+sisters&udm=2&fbs=AEQNm0Aa4sjWe7Rqy32pFwRj0UkWd8nbOJfsBGGB5IQQO6L3J5MIFhvnvU242yFxzEEp3BeeRDeomFf8DkO7myIzvXpizJSAPzgi0Ak9xb_QCAaJT8IDC8tvep3vZrJY_vsc5TQkXNvoAjWcDXYJerybE7W9fOPsM64-xdWM6PP9OnMWQNwnNPQGVVH3ugefkfmjZjQ1bEQ0Sidu07XlgXsMHHNrrMMHHw&sa=X&ved=2ahUKEwi6mYPY15-JAxVzFlkFHe8GIPsQtKgLegQIDhAB) bridges
* the funnicular railroads, in particular the [Duquesne Incline](https://www.visitpittsburgh.com/directory/duquesne-incline/)
* the city [skyline](https://www.google.com/search?sca_esv=abe1209c8be175c1&q=pittsburgh+skyline&udm=2&fbs=AEQNm0Aa4sjWe7Rqy32pFwRj0UkWd8nbOJfsBGGB5IQQO6L3J5MIFhvnvU242yFxzEEp3BeeRDeomFf8DkO7myIzvXpiPxHcR23NF3WnfxMWCBegt9ROeYDb-laChWX6YWu-EKw-o5kvXhPAVThv4hC-1g_mMpHBbTCBUQfSJCVMA8pPU9L8mnmykZF6of8yQt2bmVQB5_pI06k9ThV9tas6jY5ekYpYEg&sa=X&ved=2ahUKEwjoqs3y15-JAxVLK1kFHXfeCSEQtKgLegQIFxAB&biw=1470&bih=831&dpr=2)
* presuming the venue does end up being some combination of Pitt and CMU, there are some iconic things on the campuses as well, such as the [Cathedral of Learning](https://en.wikipedia.org/wiki/Cathedral_of_Learning), the [Pausch Bridge](https://www.colorkinetics.com/global/showcase/pauschbridge), or [The Fence](https://www.cmu.edu/stugov/fence/index.html)

**Comment by cormullion** [comment]
@rkurchin Thanks - I’ll indulge in a bit of armchair tourism! 😀

**Comment by cormullion** [comment]
Not very original, but OK-ish-looking...

 <img width="830" alt="Screenshot 2024-10-22 at 13 37 54" src="https://github.com/user-attachments/assets/ca1c7d21-8b4b-40fe-8d08-50a42a26a119">

**Comment by cormullion** [comment]
the bridges 

![square layout colour 2](https://github.com/user-attachments/assets/d0fa4d97-deee-4bcb-abd7-a17d5913db4d)

gold border needs more forging

**Comment by cormullion** [comment]
Another bridge

![bridges2-400](https://github.com/user-attachments/assets/ecb7b161-483c-4aad-8deb-e7a15d77426b)

**Comment by cormullion** [comment]
I tried something with the awesome 'cathedral' building, but not sure if it works:

 ![cathy-dot-397](https://github.com/user-attachments/assets/5206861a-bf12-4a79-a2ee-e749e6271725)

**Comment by cormullion** [comment]
Something a little more quirky:
![impossible-400](https://github.com/user-attachments/assets/038466ef-6e7d-45ec-b573-767ced742d78)

**Comment by cormullion** [comment]
a simple array 

![jcon2025-idea](https://github.com/user-attachments/assets/3799e7d4-04f4-423f-8e53-898992dcb0ab)

**Comment by rayegun** [comment]
Hey @cormullion Thank you so much for putting these together :). Asking the committee which they prefer rn (sorry for the extremely extended reply, very busy semester)!

**Comment by cormullion** [comment]
More ideas and suggestions welcome - need some more inspiration 😀

**Comment by ranjanan** [comment]
I like the one on the left in the last set too, and the golden bridge. I think making the last one gold and capitalizing the "I" will make for a great logo.

**Comment by cormullion** [comment]
![example](https://github.com/user-attachments/assets/c1971d68-d19d-44ad-8dad-b2957a1f67f7)

**Comment by rayegun** [comment]
Looks great @cormullion wonderful as always! Could I get as many templates based on that design as you like, and the logos? Similar to that image above and then also some that are like the talk advertising templates from last year!

**Comment by rayegun** [comment]
Also could we get updated social media banners based on this?

**Comment by rayegun** [comment]
Happy to hop on a zoom too @cormullion to chat about materials we'd like :)

**Comment by cormullion** [comment]
😀 Sure thing - what's the location exactly (probably need to specify?)

**Comment by rayegun** [comment]
University of Pittsburgh and CMU :)

**Comment by rayegun** [comment]
Exact building TBD

**Comment by cormullion** [comment]
If you wanted to stay with the gold/bridges idea, you could do:

![Image](https://github.com/user-attachments/assets/4b5d3777-b7ed-487c-9856-0ed549d27179)

although there's not a lot of colour.

Easily fixed by adding some more:

![Image](https://github.com/user-attachments/assets/1f3e648d-ca87-448b-80db-c6760a9ea5d8)

Some people like the REPL design - this is 1.11.5 now:

![Image](https://github.com/user-attachments/assets/33ffdfdc-0560-4289-9e15-a8dee80ecaf2)

(I"m never sure how readable  this is...)

**Comment by yangmr04** [comment]
> I tried something with the awesome 'cathedral' building, but not sure if it works:
> 
> ![cathy-dot-397](https://github.com/user-attachments/assets/5206861a-bf12-4a79-a2ee-e749e6271725)

Can we try the cathedral on t-shirt? I personally really like the icon itself and it's colorful but not intrusive.

**Comment by cormullion** [comment]
It's possible:

![Image](https://github.com/user-attachments/assets/6ac02474-e8f0-4439-86d4-586a80ebb567)

**Comment by yangmr04** [comment]
> If you wanted to stay with the gold/bridges idea, you could do:
> 
> ![Image](https://github.com/user-attachments/assets/4b5d3777-b7ed-487c-9856-0ed549d27179)
> 
> although there's not a lot of colour.
> 
> Easily fixed by adding some more:
> 
> ![Image](https://github.com/user-attachments/assets/1f3e648d-ca87-448b-80db-c6760a9ea5d8)
> 
> Some people like the REPL design - this is 1.11.5 now:
> 
> ![Image](https://github.com/user-attachments/assets/33ffdfdc-0560-4289-9e15-a8dee80ecaf2)
> 
> (I"m never sure how readable this is...)

First of all, thanks a lot! People seem to like the second one here (the spiral colors one) but want "Pittsburgh" written in there. Could you try adding the text?  

Can we also have another version with the same design but without the golden bridges (also add "Pittsburgh")? Apparently for t-shirts we are charged more by having the sixth color (the logo itself has five in there already, including white).

Could you upload svg files for both designs? Thanks!

**Comment by cormullion** [comment]
No takers for your cathedral idea? I thought it was intriguing... 🤔

So I've uploaded these two:

        juliacon2025/tshirt-a.svg
        juliacon2025/tshirt-b.svg

<img width="587" alt="Image" src="https://github.com/user-attachments/assets/2a63a26e-619e-431a-900f-2a7e517ee0ba" />

**Comment by yangmr04** [comment]
Hi @cormullion ! We decided to go with `tshirt-b`! Thank you so much for all of these! If you can't make it to JuliaCon this year, I can ship a few t-shirts to you (only if they don't run out though, it's too popular!). Thanks again!

**Comment by cormullion** [comment]
It’s an old version of Basement Grotesque (downloaded while it was still free).
